spec/syntax.md

```diff
@@ -50,6 +51,17 @@ as the *unprefixed identifier*.
 
 No whitespace is allowed between components of an `InterpolatedIdentifier`.
 
+### `InterpolatedDynamicUrl`
+
+<x><pre>
+**InterpolatedDynamicUrl**         ::= 'url(' (QuotedString | InterpolatedDynamicUnquotedUrl) ')' | InterpolatedDynamicUnquotedUrl
```

Comment:
What's up with the `| InterpolatedDynamicUnquotedUrl` at the end here?

Reply:
that's a typo, left-over from a change I made